packages/amplify-category-storage/src/index.ts

```diff
@@ -75,9 +79,37 @@ export async function add(context: any, providerName: any, service: any) {
   return providerController.addResource(context, AmplifyCategories.STORAGE, service, options);
 }
 
-export async function console(context: any) {
-  printer.info(`to be implemented: ${AmplifyCategories.STORAGE} console`);
-}
+/**
+ * Open AWS Management Console for resources of storage category.
+ */
+export const console = async (context: $TSContext): Promise<void> => {
+  const { amplify } = context;
+  const amplifyMeta = stateManager.getMeta();
+  if (!amplifyMeta.storage || Object.keys(amplifyMeta.storage).length === 0) {
+    return printer.error('Storage has NOT been added to this project.');
+  }
+
+  const nameOverrides = {
+    S3: 'S3 bucket - Content (Images, audio, video, etc.)',
+    DynamoDB: 'DynamoDB table - NoSQL Database',
+  };
+
+  const servicesMetadata = ((await import('./provider-utils/supported-services')) as $TSAny).supportedServices;
+
+  const serviceSelection = await amplify.serviceSelectionPrompt(context, categoryName, servicesMetadata, undefined, nameOverrides);
+  try {
+    const providerController = await import(`${__dirname}/provider-utils/${serviceSelection.providerName}/index`);
+    if (!providerController) {
+      printer.error('Provider not configured for this category');
+      return undefined;
+    }
+    return providerController.console(amplifyMeta, serviceSelection.providerName, serviceSelection.service);
+  } catch (err) {
+    printer.info(err.stack);
+    printer.error('There was an error trying to open the storage web console.');
+    throw err;
```

Comment:
Apologies for not catching this in the previous review. We have a new top-level error handler that will take care of printing stack traces and printing messages. This try/catch can be removed and the error handler will handle printing the message and the stacktrace

Reply:
@edwardfoyle Thank you for your review. 
I could not find where top-level error handling is. Am I missing somewhere?
Would you tell me where top-level error handling takes place?

Reply:
> Would you tell me where top-level error handling takes place?

I found the top-level error handling.
https://github.com/aws-amplify/amplify-cli/blob/38a8603db4965ec05beb2ddd1daaa3913096c9c6/packages/amplify-cli/src/index.ts#L46

Reply:
@edwardfoyle I removed try-catch to use top-level error handling.